lib/providly/provider_domains.txt

```diff
@@ -430,6 +430,7 @@ conok.com
 consultant.com
 cookiemonster.com
 cool.br
+cool.fr.nf
```

Comment:
I'm surprise to see something else than yopmail and mailinator, where does this one comes from?

Reply:
Yes they are the other alternative addresses from yopmail : see http://www.yopmail.com/

Reply:
Nice catch, thanks for the explanation :)